tests/utils/stl/test/tests.py

```diff
@@ -79,6 +79,11 @@ def configureTest(self, litConfig):
             self.isenseRspPath = tmpBase + '.isense.rsp'
             self.compileFlags.extend(['/dE--write-isense-rsp', '/dE' + self.isenseRspPath])
 
+        # TRANSITION, google/sanitizers#328: clang-cl does not support targeting /MDd or /MTd with ASan

```

Comment:
For consistency with other occurrences:
```suggestion
        # TRANSITION, google/sanitizers#328: clang-cl does not support /MDd or /MTd with ASan
```